Lib/test/test_os.py

```diff
@@ -4647,6 +4647,41 @@ def __fspath__(self):
                 return ''
         self.assertFalse(hasattr(A(), '__dict__'))
 
+    def test_fspath_set_to_None(self):
+        class Foo:
+            __fspath__ = None
+
+        class Bar:
+            def __fspath__(self):
+                return 'bar'
+
+        class Baz(Bar):
+            __fspath__ = None
+
+        good_error_msg = (
+            r"((expected)|(should be)) str, bytes or os.PathLike( object)?, not {}".format
```

Comment:
```suggestion
            r"(expected|should be) str, bytes or os.PathLike( object)?, not {}".format
```
Don't need that many parentheses